ext/standard/tests/strings/implode_error.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test implode() function: error conditions
+--FILE--
+<?php
+/* only glue */
+try {
+    var_dump( implode("glue") );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* NULL as pieces */
+try {
+    var_dump( implode("glue", NULL) );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+
+/* integer as glue */
+try {
+    var_dump( implode(12, "pieces") );
+} catch (TypeError $e) {
+    echo $e->getMessage(), "\n";
+}
+?>
+--EXPECTF--
```

Comment:
This should be `EXPECT`.

Reply:
Got it. Will change it tonight and push. Thx